services/api-server/src/simcore_service_api_server/services_http/webserver.py

```diff
@@ -185,14 +186,16 @@ async def _page_projects(
         limit: int,
         offset: int,
         show_hidden: bool,
-        search: str | None = None,
+        search_by_project_name: str | None = None,
     ) -> Page[ProjectGet]:
         assert 1 <= limit <= MAXIMUM_NUMBER_OF_ITEMS_PER_PAGE  # nosec
         assert offset >= 0  # nosec
 
         optional: dict[str, Any] = {}
-        if search is not None:
-            optional["search"] = search
+        if search_by_project_name is not None:
+            filters_dict = {"search_by_project_name": search_by_project_name}
+            filters_json = json.dumps(filters_dict)
```

Comment:
MINOR: we always use our own implementation of json.dumps in `common_library.json_serialization.json_dumps`